tests/test_otmm_makam.py

```diff
@@ -0,0 +1,697 @@
+            {
+                "mbid": "1ed6c08e-9d5b-4e5c-a0a4-41acb3445000",
+                "attribute-list": ["accordion"],
+                "type": "instrument",
+                "name": "Ceyhun Çelikten",
+            },
+            {
+                "mbid": "30380155-5974-4cf5-aed9-60371470005d",
+                "attribute-list": ["oud"],
+                "type": "instrument",
+                "name": "Yıldıran Güz",
+            },
+            {
+                "mbid": "3411c847-4801-4a19-893c-521e5719cf44",
+                "attribute-list": ["clarinet"],
+                "type": "instrument",
+                "name": "Şükrü Kabacı",
+            },
+            {
+                "mbid": "4d3ff01d-71c1-4c1c-9a44-4e4448cfb9f8",
+                "attribute-list": ["cello"],
+                "type": "instrument",
+                "name": "Özer Arkun",
+            },
+            {
+                "mbid": "5121835d-84fb-44c6-8eed-8a8da4c444ba",
+                "attribute-list": ["ney"],
+                "type": "instrument",
+                "name": "Ahmet Şahin",
+            },
+            {
+                "mbid": "64944130-34f4-43bf-914f-e741a04e8350",
+                "attribute-list": ["strings"],
+                "type": "instrument",
+                "name": "Kempa Yaylı Grubu",
+            },
+            {
+                "mbid": "70c930f4-be11-48c6-97b8-b31547a31497",
+                "attribute-list": ["viola"],
+                "type": "instrument",
+                "name": "İbrahim Şentürk",
+            },
+            {
+                "mbid": "72a54b29-9c8d-4ec7-8c12-233bb9f4551a",
+                "attribute-list": ["viola"],
+                "type": "instrument",
+                "name": "Metin Kabacı",
+            },
+            {
+                "mbid": "8c968ab9-1a75-4ad2-b4a1-faa1f51d0006",
+                "attribute-list": ["kanun"],
+                "type": "instrument",
+                "name": "Taner Sayacıoğlu",
+            },
+            {
+                "mbid": "8c98c6c1-63a8-4573-9a76-31dc41fec8ad",
+                "attribute-list": ["ney"],
+                "type": "instrument",
+                "name": "Eyüp Hamiş",
+            },
+            {
+                "mbid": "9b6e98f8-b695-45da-8c9c-68d4bab460fd",
+                "attribute-list": ["tanbur"],
+                "type": "instrument",
+                "name": "Murat Aydemir",
+            },
+            {
+                "mbid": "b5e2f977-cadd-458a-8171-cafdfe6a331c",
+                "attribute-list": ["percussion"],
+                "type": "instrument",
+                "name": "Fahrettin Yarkın",
+            },
+            {
+                "mbid": "cab02cb9-d0b2-4ba5-8ea3-e5c6ab46b005",
+                "attribute-list": ["violin"],
+                "type": "instrument",
+                "name": "Baki Kemancı",
+            },
+            {
+                "mbid": "cdc69411-25f2-4fe3-88e5-41da2b8c474b",
```

Comment:
just a nit, and up to you - for the the tests, you dont have to use the complete real data files - you can reduce them so they have e.g. just one item in each dictionary, and it will make the test code smaller

Reply:
yes, I am aware I did the same for the Saraga dataset! In this case, I noticed that there were a lot of special characters in the metadata files so I thought it wouldn't do bad to check the encoding/decoding of the json file. But I'll keep that in mind!!

Reply:
Then it might be better to have concise unittest(s) for characters specific to Turkish, namely çÇ, ğĞ, ıİ, iİ, öÖ, şŞ, üÜ. (Emphasis on the difference in the capitals of ı and I.)

Reply:
I'll introduce all these characters to the test file and see if there's any problem showing up. Thanks for the suggestion!